test/lender/Lender.sol

```diff
@@ -9,27 +9,27 @@ import './Safe.sol';
 import './Cast.sol';
 import './Element.sol';
 
+
 contract Lender {
   address public admin;
   address public illuminate; // TODO authorized setter for this
 
-  // TODO the nature of these addresses?
   address public swivelAddr; // addresses of the 3rd party protocol contracts
-  address public pendleRouter;
-  address public tempusRouter;
+  address public pendleAddr;
+  address public tempusAddr;
   address public senseToken;
 
   event Lend(uint8 principal, address indexed underlying, uint256 indexed maturity, uint256 returned);
   event Mint(uint8 principal, address indexed underlying, uint256 indexed maturity, uint256 amount);
 
   /// @param i the deployed Illuminate contract
-  constructor(address i, address s, address su, address t, address sa) {
+  constructor(address i, address s, address pr, address t, address st) {
```

Comment:
so when we can't use the single letter paradigm, we use a full word. it's a one or the other kinda thing.

so probs `(i, swivel, p, t, sense)` -- we'll usually use full words for whatever would have clashed with one letter